src/ray/raylet/scheduling/scheduling_ids.h

```diff
@@ -185,6 +185,17 @@ class ResourceID : public BaseSchedulingID<SchedulingIDTag::Resource> {
   /// "ClusterResourceSchedulerTest" have different namespaces.
   friend void SetUnitInstanceResourceIds(absl::flat_hash_set<ResourceID> ids);
 
+  bool IsPlacementGroupWildcardResource() const {
```

Comment:
I think it's better to use a global function `bool IsPlacementGroupWildcardResource(const std::string &resource_name);` to instead of putting it as a member function of ResourceID.